src/main/java/net/pms/newgui/DbgPacker.java

```diff
@@ -53,7 +56,10 @@ public DbgPacker() {
 			zippedLogFile = PMS.getConfiguration().getDefaultLogFilePath();
 		}
 		if (!zippedLogFile.isEmpty()) {
-			zippedLogFile = FileUtil.appendPathSeparator(zippedLogFile) + "ums_dbg.zip";
+			DateFormat dateFormat = new SimpleDateFormat("yyyy-MM-dd-HH:mm");
```

Comment:
Not sure if the fix for this got reverted on the way, but as @ExSport said, the colon breaks Windows support

Reply:
@SubJunk Fixed it Thanks. :smile: